src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -112,7 +112,15 @@ namespace intel_cpu {
 
 using const_node_ptr = const std::shared_ptr<const ov::Node>;
 
-bool Transformations::fuse_type_to_convert(const std::shared_ptr<ngraph::Node>& node, ov::element::Type to, size_t idx) {
+bool Transformations::fuse_type_to_convert(const std::shared_ptr<ngraph::Node>& node, const precisions_map& precisions) {
+    const auto& from = node->get_output_element_type(0);
+    auto type_pair = std::find_if(precisions.begin(), precisions.end(),
```

Comment:
As `precisions` is map maybe find function member will work (maybe cast from to Type_t to be same as key is required)?
```suggestion
    auto type_pair = precisions.find(from);
```

Reply:
done